check.go

```diff
@@ -853,31 +853,31 @@ func (runner *suiteRunner) checkFixtureArgs() bool {
 }
 
 func (runner *suiteRunner) reportCallStarted(c *C) {
-	runner.reporter.StartTest(c)
+	runner.reporter.Report(startTest, c)
```

Comment:
It seems that the interface definition has not been changed, it has yet the old methods and `Report` has not been added.